common/src/main/java/com/sequenceiq/cloudbreak/logger/LoggerConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.logger;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class LoggerConfiguration {
+
+    private LoggerConfiguration() {
+    }
+
+    public static Boolean isJsonFormatEnabled() {
+        return Boolean.parseBoolean(getProperty("logger.format.json.enabled", null));
+    }
+
+    public static String getJsonFormatMdcName() {
+        return getProperty("logger.format.json.mdc.name", "context");
+    }
+
+    public static Integer getSplitterMaxChunLength() {
+        return Integer.parseInt(getProperty("logger.appender.splitter.max.chunk.length", "12000"));
+    }
+
+    private static String getProperty(String property, String defaultValue) {
+        String envProperty = property.toUpperCase().replace(".", "_");
```

Comment:
I don't really get this. so there are 3 properties above, you wrote them with lowercase and dot separated and right after you convert them. wouldn't be it easier just to have them written the way we expect them?

Reply:
we use this both as a system property + environment variable, first we are checking the environment property, lower we are simply checking the system property (system property used mostly locally, env vars are used for qa/mow deployments), we can do the opposite here, but I think it does not really matter.

Reply:
That's why we should use Spring instead of implementing a Spring feature this way....

Reply:
sry, it is not supported by spring